content/23.vaccines-novel.md

```diff
@@ -108,7 +122,7 @@ The shift towards omics-based approaches to vaccine development began to take ho
 In this way, the genomic revolution catalyzed a fundamental shift in the development of vaccines.
 Such technologies hold the potential to revolutionize the role of vaccines, given their potential to address one of the major limitations of vaccines today and facilitate the design of therapeutic, rather than just prophylactic, vaccines [@doi:10.3390/vaccines7020037].
 
-Nucleic-acid based approaches share an underlying principle: a vector that delivers the information needed to produce an antigen can trigger an immune response without introducing an infectious agent.
+Nucleic-acid based approaches share an underlying principle: a vector that delivers the information needed to produce an antigen can trigger an immune response without introducing an infectious agent.<!--I'm not sure what this means? So viral proteins introduced into our system can both be protective but make us sick. While allowing our cells to generate these viral proteins doesn't make us sick somehow?-->
```

Comment:
```suggestion
Nucleic-acid based approaches share an underlying principle: a vector that delivers the information needed to produce an antigen can trigger an immune response.
```
Since LAV are in the other manuscript, let's just drop that part!

Reply:
Here I was thinking it would be helpful to have a single sentence that gave a bit of an intuition for how these nucleic acid approaches reduce the risk by allowing our cells to generate the viral antigens as opposed to introducing the antigen. But I'll defer to @rando2 about where this text makes most sense, since you are much more aware of all the different papers and what they cover.

Reply:
Thank you @ajlee21! I updated the "suggestion" in the comment above.

Reply:
I think that's great! Thanks for the addition. So even compared to subunit vaccines this nuclei acid vaccine that is generating the same protein, is still safer?

Reply:
I'm not sure about that! I think the safety concerns are mainly with LAV (and have largely been mitigated today anyways, but there are some issues, like the current polio case in the news that I believe was caused by an unvaxxed person interacting with someone who had recently received a LAV)